### ISSUE TITLE
Fast Forward?

### ISSUE THREAD

**Issue by Herdna** [MGraefe/RP-Soundboard#94]
Hi it whould be real nice with a fast forward or fast backward function so that you dont have to play the a mix from teh very beginning. and also a counter that show where in a mix or song you are like 0:22 - 3:23. :)

**Comment by MGraefe** [comment]
You can achieve most of that with the crop feature. Right click a button and click advanced.

**Comment by Herdna** [comment]
> You can achieve most of that with the crop feature. Right click a button and click advanced.

yeah i know but thats only when u want too choose a starting point of the track. i mean u should be able to fast forward in the track while ur in a game. its too much work to crop while ur playing you know :P
